grafana/resource_builtin_role_assignment.go

```diff
@@ -225,7 +224,7 @@ func createOrRemove(meta interface{}, name string, changes []RoleChange) error {
 			err = client.DeleteBuiltInRoleAssignment(br)
 		}
 		if err != nil {
-			return errors.New(fmt.Sprintf("Error with %s %v", name, err))
+			return fmt.Errorf("Error with %s %v", name, err)
```

Comment:
Nit: when wrapping an `error` use `%w` instead of `%v`, so it works with `errors.Unwrap`.